docs/monitoring/spm-faq.md

```diff
@@ -144,8 +155,15 @@ there is still a way to monitor your service. If you are OK with
 installing SPM client on separate linux box, and pointing it (more about
 that further below) to your service (which should be monitored) running
 on Windows or Mac, you can use Sematext to monitor all non-OS related
-metrics. That means you will not be able to see CPU or system Memory
-metrics, but metrics specific to your process will be displayed (for
+metrics. You can stop and disable OS metric collector using 
+
+```bash
+sudo service spm-monitor-sta stop 
```

Comment:
We are exposing "sta" as the name to users?
Maybe we need to think about this one before we make that public.  Would "infra" be better?  I saw we also made references to "Infra agent" elsewhere in this doc.